### ISSUE TITLE
On Potential Data Leakage in the Evaluation Protocol

### ISSUE THREAD

**Issue by gsd110** [XJTU-EEG/LibEER#12]
Thank you to the authors for your interesting and valuable work.

In the paper, it is stated: "In scenarios where a validation set is not available, LibEER identifies the optimal performance on the testing set during training as the final outcome for that training iteration."

My question is: Doesn't using the test set to select the optimal checkpoint during training introduce data leakage? This seems to use the test set for model selection, which could lead to an over-optimistic estimate of the final performance.
